pom.xml

```diff
@@ -27,4 +27,26 @@
 		<module>authentication_lib</module>
 		<module>authentication_ui_bootstrap5</module>
 	</modules>
+
+	<distributionManagement>
+    	<repository>
+        	<id>github</id>
+        	<name>GitHub Packages</name>
+        	<url>https://maven.pkg.github.com/OCTRI/authentication-lib</url>
+    	</repository>
+	</distributionManagement>
+
+	<build>
```

Comment:
I came across this bit to automate package uploading and publishing to GitHub Packages or maven. Wasn't sure if this is useful in our use case, but still added it. If it seems like an overkill, then can remove it.

Reply:
I'm not certain, but I don't think that we specifically need this. We're inheriting `maven-deploy-plugin` from the Spring Boot parent POM. Let's delete the `<build>` section and see how it goes.

Reply:
I tested deploying the last three versions this morning, and it worked without this `<build>` section.